### DISCUSSION TITLE
What are the steps to contribute?

### DISCUSSION THREAD

**Discussion by Hetarth02** [laravel/framework#41676] [Q&A]
Hello,

I might have one or more array helper functions in mind and I have also code implementation ready for one of them.

I wanted to know that where can I propose to integrate my code in the framework. What are the steps I should follow before opening a direct PR?

I asked this here after reading the Contribution Guide in the laravel documentation.

**Comment by Raza9798** [comment]
I found all the PR are pointed to this branch [10.x](https://github.com/laravel/framework/tree/10.x)
